x/bank/simulation/operations.go

```diff
@@ -22,6 +21,8 @@ const (
 	OpWeightMsgMultiSend      = "op_weight_msg_multisend"
 	DefaultWeightMsgSend      = 100 // from simappparams.DefaultWeightMsgSend
 	DefaultWeightMsgMultiSend = 10  // from simappparams.DefaultWeightMsgMultiSend
+
+	DistributionModuleName = "distribution"
```

Comment:
```suggestion
	distributionModuleName = "distribution"
```
Seems like this is used only in this package?